### ISSUE TITLE
Heredoc renders "<<EOT" and "EOT"

### ISSUE THREAD

**Issue by felixhummel** [hashicorp/terraform#27162]
### Terraform Version
```
Terraform v0.14.0
```

### Terraform Configuration Files
See https://github.com/felixhummel/terraform-output-eot (includes steps, expected and actual)

### Expected Behavior
prints
```
foo
bar
```

### Actual Behavior
prints
```
<<EOT
foo
bar

EOT
```

### Steps to Reproduce
1. `terraform init`
2. `terraform apply`
3. `terraform output foo`

**Comment by alisdair** [comment]
This is an intentional change. See #27045 and #27039 for the details.

If you're writing scripts relying on `terraform output`, we recommend using the `-json` flag. The JSON format is stable, whereas the human readable output is subject to change.

**Comment by alisdair** [comment]
Hi @felixhummel. On rereading my previous comment it was more curt than I'd intended, so I'm sorry about that!

I also have some news that may be of interest to you. We just merged #27212, which adds a `-raw` flag to `terraform output` that hopefully helps this use case. Your reproduction case with a build from the main branch:

```shellsession
$ tf output foo
<<EOT
foo
bar

EOT
$ tf output -raw foo
foo
bar
```

This change is scheduled to ship in the next 0.14 patch release.

**Comment by felixhummel** [comment]
Hi @alisdair. Thanks! I like the `-raw` flag. This makes scripting easier than piping to jq. :)
